doc/overlays.xml

```diff
@@ -63,31 +63,31 @@ a symbolic link to it in <filename>~/.config/nixpkgs/overlays/</filename> direct
 packages.</para>
 
 <programlisting>
-self: super:
+pkgsself: pkgssuper:
 
 {
-  boost = super.boost.override {
-    python = self.python3;
+  boost = pkgssuper.boost.override {
+    python = pkgsself.python3;
   };
-  rr = super.callPackage ./pkgs/rr {
-    stdenv = self.stdenv_32bit;
+  rr = pkgssuper.callPackage ./pkgs/rr {
+    stdenv = pkgsself.stdenv_32bit;
   };
 }
 </programlisting>
 
-<para>The first argument, usually named <varname>self</varname>, corresponds to the final package
+<para>The first argument, usually named <varname>pkgsself</varname>, corresponds to the final package
```

Comment:
It is not, and while I think `self` should probably be named `fixed` in the long run, adding the `pkgs` prefix becomes quite verbose.